src/modules/game.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable */
+// Game
```

Comment:
- [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the eslint disabling command and please fix all the linters error 🙏